__tests__/app.test.js

```diff
@@ -56,4 +56,24 @@ describe('GET Categories', () => {
             });
         })
     });
+});
+
+describe('GET comments by review ID', () => {
+    test('Should return all comments of a given review', () => {
+        return request(app).get('/api/reviews/3/comments')
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments;
+            comments.forEach(comment => {
+                expect(comment).toHaveProperty('comment_id');
+                expect(comment).toHaveProperty('body');
+                expect(comment).toHaveProperty('review_id');
+                expect(comment).toHaveProperty('author');
+                expect(comment).toHaveProperty('votes');
+                expect(comment).toHaveProperty('created_at');
+            });
```

Comment:
I like that you are asserting for each property separately here. You are however only asserting for the existence of each property but they could be any value at all and still pass the test.
You should be able to assert at least the data type of each property - but specifically for the review_id (3 in this case).
Also, it's crucial that you assert for the length of the array - otherwise an empty array would still pass